### ISSUE TITLE
[Feature request]: Print Category feature

### ISSUE THREAD

**Issue by carlosvirreira** [Shelf-nu/shelf.nu#399]
### Is your feature request related to a problem? Please describe?

Hi Carlos, hope you are doing well! I was looking to see about a "print category" feature that might help us out a bit. So what REDACTED and I are being told from our boss is that he wants a print out of everything in certain categories. 

### Describe the solution you'd like

No pictures, just a list of names and the location.

**Comment by carlosvirreira** [comment]
@jurrejansen maybe this can be achieved with the same functionality as the reports?

**Comment by jurrejansen** [comment]
@carlosvirreira yes, while adding assets to a batch they could filter on 'category' and fill up the batch with all assets that have that category.

https://www.figma.com/file/ykvc0MhPeNLsrJ4tJsKosQ/Shelf-%7C-UX-Scenarios?type=design&node-id=9-2&mode=design

Not sure is meant with "No pictures, just names and the location"

**Comment by carlosvirreira** [comment]
Ok. I am not sure what the batches is maybe you can describe it on another issue. For now I will instruct him to do an export, to sort the CSV and get his data like this.

**Comment by jurrejansen** [comment]
Batches is a concept I was working on that enables a user to create a batch entry, add assets, customize the QR tag (that will apply for the whole batch) and then either run a print command, but more realistically, download the customized QR files for all added assets in 1 click.

https://www.figma.com/file/ykvc0MhPeNLsrJ4tJsKosQ/Shelf-%7C-UX-Scenarios?type=design&node-id=9-2&mode=design

This will enable the user to add all assets that belong to al certain category and then download / print them. However, I now suspect what was described in your original message is: Filtering the asset index by a certain category and then generating a PDF of the filtered index.

I'll make a new issue for Printing / Batches

**Comment by DonKoko** [comment]
I was just looking through some issues and found this. I think this can be very easily achieved, we can do the same as what supabase does. It allows you to download a table's content, to a csv or just a text file. So if they want to download all from a specific category, they can just filter by that category. Keep in mind this doesnt export all asset data, just flat simple data, kinda like the table itself.

**Comment by sbe-visma** [comment]
Hi,

I'd also like this but for kits and with images, it could in theory be a report but it'd be nice with a shortcut from the kit page to for something like "print pick list".

With inexperienced teardown crews it'd be really helpful to be able to give them a list with photos, names, and ID:s of what they need to pack, and the name of the kit - potentially with checkboxes at the end to allow pepole to check of that they've packed it back into the kit on site.
I've done manual versions of this, but a quick, PDF export would be a fantastic timesaver
